rest-api/etl/source_to_concept_map.csv

```diff
@@ -1,3 +1,5 @@
+SexAtBirth_Male|1585846|ppi-sex|MALE|8507|Gender|1970-1-1|2099-12-31|\N|1
+SexAtBirth_Female|1585847|ppi-sex|FEMALE|8532|Gender|1970-1-1|2099-12-31|\N|1
```

Comment:
Do we need to worry about SexAtBirth_Intersex and SexAtBirth_SexAtBirthNoneOfThese?